### PR TITLE
Added documentation augmented client return urls.

### PR DESCRIPTION
**Pull request by davidv1992**
Documentation for irmago pull request https://github.com/privacybydesign/irmago/pull/126

### CODE REVIEW

**Review by sietseringers — CHANGES_REQUESTED** [review]

File: docs/irma-server.md [hunk 1]
```diff
@@ -162,6 +162,10 @@ The global options also work when `no_auth` is enabled. Thus in this case a sess
 
 In development mode, when `production` is `false`, the defaults for `disclose_perms`, `sign_perms` and `issue_perms` are `["*"]`. In order to protect any IRMA private keys that the server has access to from unintended use by others, when `production` is true the default of `issue_perms` is `[]`: no one can issue unless the global `issue_perms` is modified or unless specific requestors receive nonempty `issue_perms`.
 
+### Augmented client return urls
```
[the remark this answers is not in the document]
Comment (davidv1992) [reply docs/irma-server.md:169]: fixed

File: docs/irma-server.md [hunk 2]
```diff
@@ -162,6 +162,10 @@ The global options also work when `no_auth` is enabled. Thus in this case a sess
...
+
+The server can be configured to [augment the client return url](Augmenting the client return URL) under certain circumstances. In order to enable this feature, the `augment_client_return_url` setting needs to be set to `true`.
```
[the remark this answers is not in the document]
Comment (davidv1992) [reply docs/irma-server.md]: fixed

File: docs/session-requests.md [hunk 3]
```diff
@@ -373,6 +373,29 @@ This URL does not necessarily have to be a web URL; it can also be a universal l
 
 On iOS, toggling back to the calling app or website after the session can be simulated by navigating towards the calling app using a `clientReturnUrl`, either a normal URL or universal link. This works on Android as well, so currently this is the only platform-independent way of ensuring that the used ends up at a specified place after the session. If the URL opens a website, however, then the user is navigated towards a new browser tab instead of back to an existing browser tab, so in website-IRMA-website flows you will need to reload your webapp and state in the newly opened tab.
 
+### Augmenting the client return URL
+
+It is possible to have the irma server augment the `clientReturnUrl` with the server token of a session. This token is then appended as a query parameter with name token to the `clientReturnUrl`. To enable this, both the server configuration flag augment-client-return-url needs to be enabled, as well as the `augmentReturnUrl` session request parameter needs to be true
```
[the remark this answers is not in the document]
Comment (davidv1992) [reply docs/session-requests.md]: fixed

File: docs/session-requests.md [hunk 4]
```diff
@@ -373,6 +373,29 @@ This URL does not necessarily have to be a web URL; it can also be a universal l
...
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Session request (JSON)-->
+```json
+{
+  "@context": "https://irma.app/ld/request/disclosure/v2",
+  "disclose": [
+    ...
+  ],
+  "clientReturnUrl": "https://privacybydesign.foundation"
```
[the remark this answers is not in the document]
Comment (davidv1992) [reply docs/session-requests.md]: Fixed (url also changed in preexisting documentation!)

File: docs/irma-server.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidv1992) [reply docs/irma-server.md:169]: Fixed

File: docs/irma-server.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (davidv1992) [reply docs/irma-server.md]: Fixed

File: docs/session-requests.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (davidv1992) [reply docs/session-requests.md]: fixed

File: docs/session-requests.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (davidv1992) [reply docs/session-requests.md]: Changed url (also in previous example which was source for this one)

fixed the bug.

**Pull request merged**